### PR TITLE
Feat: parse language using medline_parser

### PR DESCRIPTION
**Pull request by JovanVeljanoski**
Enables the parsing of the language of an article.

- [x] Tests pass
- [ ] Review

### CODE REVIEW

File: pubmed_parser/medline_parser.py [hunk 1]
```diff
@@ -549,6 +549,9 @@ def parse_article_info(
     journal = article.find("Journal")
     journal_name = " ".join(journal.xpath("Title/text()"))
 
+    language = article.findall("Language")
+    language = [''.join(elem.itertext()) for elem in language]
```
Comment (itallix) [inline pubmed_parser/medline_parser.py:553]: let's do one liner and avoid variable reassigning?

Comment (JovanVeljanoski) [reply pubmed_parser/medline_parser.py:553]: Yeah, i was not sure if we want to do that, or optimize for readability :). 
I am happy either way, I can make it one line

Comment (itallix) [reply pubmed_parser/medline_parser.py:553]: Yeah, it's cleaner to have one line or use different variable names for element and it is text value like with journal.

Comment (JovanVeljanoski) [reply pubmed_parser/medline_parser.py]: Ok, i've decided on two separate variable names. Do you agree?

**Comment by itallix** [comment]
LGTM!

**Pull request merged**